src/main/java/gregtech/api/recipes/FluidCellInput.java

```diff
@@ -56,15 +42,10 @@ public static ItemStack getFilledCell(Fluid fluid) {
     }
 
     @Override
-    public boolean apply(@Nullable ItemStack itemStack) {
+    public boolean acceptsStack(@Nullable ItemStack itemStack) {
         IFluidHandlerItem stackFluid = itemStack.getCapability(CapabilityFluidHandler.FLUID_HANDLER_ITEM_CAPABILITY, null);
```

Comment:
null check `itemStack` before trying to get the capability? Since it is nullable.